### DISCUSSION TITLE
Location LDAP Search OU Not Applying

### DISCUSSION THREAD

**Discussion by Zomagoras** [snipe/snipe-it#9377] [Q&A] [answered]
Hello,

I am running into issues with syncing users via LDAP. I have the LDAP integration properly set up and configured, but when I run an LDAP sync on a specific location that should only sync users from a set OU it just syncs from the main LDAP filter and ignores the locations filter. The sync does succeed, but as mentioned it appears to ignore the locations LDAP search OU and just go off the filter set via the LDAP configuration.

Main LDAP config:
![image](https://user-images.githubusercontent.com/49960343/113197775-f0fecc00-9219-11eb-9361-647151035d86.png)

Location 1 Example:
![image](https://user-images.githubusercontent.com/49960343/113197874-17246c00-921a-11eb-858c-c0272f3da9b7.png)

I have 5 locations set up like this, but with slightly different filters that are supposed to sync there users from within. (Info changed for confidentiality)
All the filters roughly look like this, but with the city changing: (OU=Org Users,OU=Users,OU=North America,OU=City,DC=company,DC=com)

**Comment by Zomagoras** [comment, marked as answer]
Solved this issue - The formatting of the LDAP Search OU acts differently in the location section it seems.

What I originally had: (OU=Org Users,OU=Users,OU=North America,OU=City,DC=company,DC=com)
What worked: (OU=City,OU=North America,OU=Users,OU=Org Users,DC=company,DC=com)

**Reply by Toby737** [reply]
Hi there,
Just to get my head around this.
So on your AD server, your users are inside the container Org Users?
